starknet_py/tests/e2e/docs/code_examples/test_contract_function.py

```diff
@@ -2,12 +2,23 @@
 import pytest
 
 from starknet_py.contract import Contract, ContractFunction
+from starknet_py.net.client_models import ResourceBounds
 
 
-def test_prepare(map_contract: Contract):
-    # docs-start: prepare
-    prepared_function_call = map_contract.functions["put"].prepare(key=10, value=20)
-    # docs-end: prepare
+def test_prepare_invoke_v1(map_contract: Contract):
```

Comment:
Should we also add `test_prepare_call`?

Reply:
I intentionally left out `test_prepare_call` because I think `prepare_call` isn't generally very useful. However I didn't want to completely remove it from the codebase. There are examples for `ContractFunction.call` and `ContractFunction.call_raw`, which should be sufficient.

Reply:
Makes sense